www/js/app.js

```diff
@@ -1635,27 +1629,71 @@ document.getElementById('archiveFilesLbl').addEventListener('keydown', function
     }
 });
 
+/** Drag and Drop handling for ZIM files */
+
+// keep track of entrance event so we only fire the correct leave event 
+var enteredelement;
```

Comment:
For variables we use camelCase to make more readable. Change to `enteredElement`.